starboard/nplb/player_get_audio_configuration_test.cc

```diff
@@ -0,0 +1,219 @@
+// Copyright 2023 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <vector>
+
+#include "starboard/nplb/player_test_fixture.h"
+#include "starboard/string.h"
+#include "testing/gtest/include/gtest/gtest.h"
+
+#if SB_API_VERSION >= 15
+
+bool operator==(const SbMediaAudioConfiguration& left,
```

Comment:
nit: can this be moved into the anonymous namespace?

Reply:
If moving it to anonymous namespace, std cannot find the operator==.